projects/addon-table/components/table/td/td.style.less

```diff
@@ -31,6 +35,12 @@
     }
 }
 
+@keyframes repaint {
+    to {
+        border-right-color: var(--tui-base-04);
```

Comment:
People might want to override this. Can we do something like `isolation: isolate` here? Or some other harmless rule. We need the same in `th` too.

Reply:
I tried this, but it's doesn't work. I'm gonna try to find another solution, but if i won't be able to, we can just add a css variable for table border-color and use it in animation. What do you think?